apps/src/music/blockly/FieldChord.ts

```diff
@@ -52,6 +58,14 @@ export default class FieldChord extends Field {
     if (this.borderRect_) {
       this.borderRect_.classList.add('blocklyDropdownRect');
     }
+
+    this.backgroundElement = GoogleBlockly.utils.dom.createSvgElement(
```

Comment:
Minor - taking a look at the docs for [createSvgElement](https://developers.google.com/blockly/reference/js/blockly.utils_namespace.dom_namespace.createsvgelement_1_function.md), I think you should be able to parameterize the function call with `SVGGraphicsElement` since it extends SVGElement. Then we don't need to cast this. e.g. `this.backgroundElement = GoogleBlockly.utils.dom.createSvgElement<SVGGraphicsElement>(...);`

Reply:
TIL.  Nice.